cypress/integration/dryRunner/earyn.js

```diff
@@ -0,0 +1,46 @@
+import s from './selectors'
+
+const EARYN_DECK_ID =
```

Comment:
```suggestion
const DECK_ID =
```

We’re in the file for Earyn, so the prefix is redundant.